punx/nxdl_manager.py

```diff
@@ -38,7 +44,7 @@
 
 class NXDL_Manager(object):
 
-    """the NXDL classes found in ``nxdl_dir``"""
+    """Python access to the NXDL classes found in ``nxdl_dir``."""
```

Comment:
This docstring is insufficient. The `classes` attribute, which seems to be important in later tests should be mentioned and described because it is part of the public API.

nxdl_file_set is a path, and nxdl_defaults is an NXDL_Summary.

Reply:
adding content